config/config_test.go

```diff
@@ -65,6 +70,7 @@ host: prebid-server.prebid.org
 port: 1234
 admin_port: 5678
 default_timeout_ms: 123
+ad_server_currency: usd
```

Comment:
my implementation compares the currency strings as case insensitive, in case the request contains something like "usd" instead of "USD". you think we should remove this and just assume the strings will be uppercasse?

Reply:
I don't have a strong opinion here.

In general it's better to be very strict about input and returning good clear error messages that tell clients exactly what's wrong. For example: `"usd" is not an ISO 4217 currency code. Did you mean "USD"?`.

But... I doubt it'll matter much here. We'd only get into real trouble if there existed some currency code where both "xxx" and "XXX" were valid... but I don't think the ISO standard would ever let that happen.

Reply:
good point. although after thinking about this some more I decided to make it case sensitive (alpha) ISO codes. will update this to USD.